### PR TITLE
Join Pre-Depends with Depends

### PR DESCRIPTION
**Pull request by dimkr**
This fixes a bug in 0setup's automatic dependency resolution feature: coreutils depends on libattr1, but it's only in the Pre-Depends list, so the build is broken.

~Alternatively, this can be fixed in 0setup, by unifying Pre-Depends with Depends when generating /tmp/woof-debdb.in.~

~EDIT: trying this approach in 0setup too, to see if it's cleaner than this solution.~ Too slow when done using a shell script

@01micko Any reason not to do this? None of the dependencies are dangerous.

```
$ grep ^Pre-Depends /tmp/woof-debdb.in | sed -E 's~ \([^)]+\)~~g' | sort | uniq
Pre-Depends|adduser
Pre-Depends|adduser, debconf, mariadb-common
Pre-Depends|adduser, dpkg
Pre-Depends|adduser, init-system-helpers
Pre-Depends|adduser, udev makedev, init-system-helpers
Pre-Depends|awk
Pre-Depends|bash
Pre-Depends|ca-certificates
Pre-Depends|ceph-common
Pre-Depends|dash
Pre-Depends|debconf
Pre-Depends|debconf cdebconf
Pre-Depends|debconf debconf-2.0
Pre-Depends|debconf debconf-2.0, adduser, update-inetd
Pre-Depends|debconf debconf-2.0, init-system-helpers
Pre-Depends|debconf debconf-2.0, iso-codes, isoquery, adduser
Pre-Depends|debconf, dpkg
Pre-Depends|debconf, init-system-helpers
Pre-Depends|default-mysql-client mysql-client
Pre-Depends|dpkg
Pre-Depends|dpkg, init-system-helpers
Pre-Depends|dpkg, pypy3-lib, libbz2-1.0, libc6, libexpat1, libffi7, libgcc-s1, libgdbm6, liblzma5, libncursesw6, libsqlite3-0, libssl1.1, libtinfo6, zlib1g
Pre-Depends|dpkg, uim-data
Pre-Depends|init-system-helpers
Pre-Depends|init-system-helpers, adduser
Pre-Depends|init-system-helpers, debconf debconf-2.0
Pre-Depends|init-system-helpers, dpkg
Pre-Depends|init-system-helpers, openssl
Pre-Depends|inn2-inews
Pre-Depends|iptables
Pre-Depends|libacl1, libattr1, libc6, libgmp10, libselinux1
Pre-Depends|libacl1, libc6, libselinux1
Pre-Depends|libaudit1, libblkid1, libc6, libcap-ng0, libcrypt1, libmount1, libpam0g, libselinux1, libsmartcols1, libsystemd0, libtinfo6, libudev1, libuuid1, zlib1g
Pre-Depends|libaudit1, libc6, libcrypt1, libdb5.3, libnsl2, libpam0g, libselinux1, libtirpc3, debconf debconf-2.0, libpam-modules-bin
Pre-Depends|libaudit1, libc6, libcrypt1, libpam0g, libpam-runtime, libpam-modules
Pre-Depends|libblkid1, libc6, libcom-err2, libext2fs2, libss2, libuuid1
Pre-Depends|libblkid1, libc6, libgcrypt20, liblz4-1, liblzma5, libselinux1, libzstd1
Pre-Depends|libblkid1, libc6, libmount1, libsmartcols1
Pre-Depends|libbz2-1.0, libc6, liblzma5, libselinux1, zlib1g
Pre-Depends|libc6
Pre-Depends|libc6, libcrypt1, dpkg
Pre-Depends|libc6, libgcrypt20, liblz4-1, liblzma5, libzstd1
Pre-Depends|libc6, libgmp10, libmpfr6, libreadline8, libsigsegv2
Pre-Depends|libc6, libpcre3
Pre-Depends|libc6, libselinux1
Pre-Depends|libc6, libsmartcols1, libtinfo6
Pre-Depends|libc6, libsystemd0
Pre-Depends|libc6, libtinfo6
Pre-Depends|libc6, libtinfo6, libtinfo6
Pre-Depends|libc6, zlib1g
Pre-Depends|libpam-ldapd, libnss-ldapd, libpam-modules, inetutils-inetd
Pre-Depends|libpaper-utils
Pre-Depends|libuuid1, init-system-helpers
Pre-Depends|lsb-release
Pre-Depends|mira-assembler
Pre-Depends|mksh
Pre-Depends|mono-runtime-common
Pre-Depends|openstack-cloud-identity
Pre-Depends|openstack-cloud-services
Pre-Depends|openstack-compute-node, openstack-proxy-node
Pre-Depends|openvswitch-switch
Pre-Depends|perl-base
Pre-Depends|perl-base, perlapi-5.32.0
Pre-Depends|php-common
Pre-Depends|psmisc
Pre-Depends|pypy-lib, libbz2-1.0, libc6, libexpat1, libffi7, libgcc-s1, libgdbm6, libncursesw6, libsqlite3-0, libssl1.1, libtinfo6, zlib1g
Pre-Depends|python2-minimal
Pre-Depends|python3
Pre-Depends|python3-dicoclient
Pre-Depends|python3-minimal
Pre-Depends|python3.9-minimal
Pre-Depends|sensible-utils
Pre-Depends|skkdic
Pre-Depends|systemd
Pre-Depends|systemd-sysv sysvinit-core
Pre-Depends|trafficserver
Pre-Depends|uim-data
Pre-Depends|x11-common
Pre-Depends|x2goclient
```

### CODE REVIEW

**Review by 01micko — COMMENTED** [review]
Is the musl instructions still valid? Can both be used with 'docker' as the recommendation?

**Comment by 01micko** [comment]
I can't see a problem. I'll give it better consideration this weekend.

**Comment by dimkr** [comment]
> Is the musl instructions still valid? Can both be used with 'docker' as the recommendation?

The musl build instructions will work, but I used my toolchains (https://github.com/dimkr/toolchains) to build new binaries. I'll open a woof-CE PR soon. 

EDIT: https://github.com/puppylinux-woof-CE/woof-CE/pull/2334

File: pkg/w_apps_static/w_apps/debdb2pupdb.c [hunk 1]
```diff
@@ -9,6 +9,10 @@
  * musl-gcc -static -O3 -fomit-frame-pointer -ffunction-sections -fdata-sections -fmerge-all-constants -Wl,--sort-common -Wl,-gc-sections -o debdb2pupdb debdb2pupdb.c
```
Comment (01micko) [inline pkg/w_apps_static/w_apps/debdb2pupdb.c:9]: Thanks!

**Review by 01micko — APPROVED** [review]

**Pull request merged**